crypto/ec/ec_key.c

```diff
@@ -1112,19 +1127,61 @@ static int ecdsa_keygen_pairwise_test(EC_KEY *eckey, OSSL_CALLBACK *cb,
     OSSL_SELF_TEST_onbegin(st, OSSL_SELF_TEST_TYPE_PCT,
                            OSSL_SELF_TEST_DESC_PCT_ECDSA);
 
-    sig = ECDSA_do_sign(dgst, dgst_len, eckey);
-    if (sig == NULL)
+    // GROUP NAME param
```

Comment:
No c++ comments please..

Reply:
Sure, but also none of this works at verify =( so i don't know if I am encoding private key wrong or something like this.

Reply:
Maybe it doesnt like the public key format?
I would just try writing a simple test that does the keygen  + sign + verify and make sure that works first.. (Maybe swap to the demos folder to do that quickly), and then work backwards from there...